go-redis/plugin/tracing/tracing.go

```
@@ -0,0 +1,150 @@
+		return fmt.Errorf("tracing: unsupported redis client type: %T", rd)
+	}
+}
+
+var _defaultAttributes = []attribute.KeyValue{attribute.DBSystem("redis")}
+
+type tracingHook struct {
+	spanAttrs        []attribute.KeyValue
+	spanStartOptions []tracer.StartSpanOption
+
+	showConnString bool
+	withVariable   bool
```

golangci-lint (1.62.2)

field `withVariable` is unused

(unused)